### ISSUE TITLE
Devil May Cry Special Edition DirectX10 error

### ISSUE THREAD

**Issue by romulasry** [doitsujin/dxvk#579]
### Software information
Devil May Cry Special Edition DX10 app

### System information
- GPU: AMD RX 480
- Driver: Mesa 18.1.5
- Wine version: 3.14
- DXVK version: 0.70

### Log files
```
wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"
[0818/181517.859:INFO:crash_reporting.cc(215)] Crash reporting enabled for process: browser
[0818/181517.961:ERROR:network_change_notifier_win.cc(157)] WSALookupServiceBegin failed with: 8
[0818/181518.347:INFO:crash_reporting.cc(215)] Crash reporting enabled for process: gpu-process
[0818/181522.694:INFO:crash_reporting.cc(215)] Crash reporting enabled for process: renderer
[0818/181523.695:ERROR:gpu_process_transport_factory.cc(1029)] Lost UI shared context.
[0818/181524.062:INFO:crash_reporting.cc(215)] Crash reporting enabled for process: renderer
[0818/181524.485:ERROR:command_buffer_proxy_impl.cc(133)] ContextResult::kTransientFailure: Failed to send GpuChannelMsg_CreateCommandBuffer.
[0818/181525.458:WARNING:cert_verify_proc_win.cc(101)] Unknown error -2146762482 mapped to net::ERR_FAILED
info:  Game: DevilMayCry4SpecialEdition.exe
info:  DXVK: v0.70
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_physical_device_properties2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
info:  AMD RADV POLARIS10 (LLVM 6.0.1):
info:    Driver: 18.1.5
info:    Vulkan: 1.1.70
info:    Memory Heap[0]: 
info:      Size: 7936 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 256 MiB
info:      Flags: 0x1
info:      Memory Type[2]: Property Flags = 0x7
info:    Memory Heap[2]: 
info:      Size: 8192 MiB
info:      Flags: 0x0
info:      Memory Type[1]: Property Flags = 0x6
info:      Memory Type[3]: Property Flags = 0xe
warn:  DXGI: VK_FORMAT_D24_UNORM_S8_UINT -> VK_FORMAT_D32_SFLOAT_S8_UINT
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_10_0
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_10_0
info:  Enabled device extensions:
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_dedicated_allocation
info:    VK_KHR_descriptor_update_template
info:    VK_KHR_get_memory_requirements2
info:    VK_KHR_image_format_list
info:    VK_KHR_maintenance1
info:    VK_KHR_maintenance2
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_shader_draw_parameters
info:    VK_KHR_swapchain
warn:  DxgiSwapChain::QueryInterface: Unknown interface query
warn:  94d99bdb-f1f8-4ab0-b236-7da0170edab1
warn:  DXGI: MakeWindowAssociation: Ignoring flags
info:  DxgiVkPresenter: Recreating swap chain: 
  Format:       VK_FORMAT_B8G8R8A8_UNORM
  Present mode: VK_PRESENT_MODE_IMMEDIATE_KHR
  Buffer size:  1280x720
err:   D3D11: Cannot create texture:
  Format:  VK_FORMAT_UNDEFINED
  Extent:  262144x1x1
  Samples: 1
  Layers:  1
  Levels:  2
  Usage:   3
[0818/181530.279:INFO:crash_reporting.cc(215)] Crash reporting enabled for process: renderer
AL lib: (EE) ReleaseThreadCtx: Context 0xf6e28630 current for thread being destroyed, possible leak!
Setting breakpad minidump AppID = 329050
```

**Comment by doitsujin** [comment]
Please upload an apitrace, I can't do anything without being able to reproduce it.

**Comment by romulasry** [comment]
Is there an easy way to make an apitrace for DX10? https://github.com/doitsujin/dxvk/wiki/Common-issues#apitrace Isn't working for me.

**Comment by romulasry** [comment]
I did it but it crashes now:

[0820/191013.906:ERROR:network_change_notifier_win.cc(157)] WSALookupServiceBegin failed with: 8
[0820/191014.293:INFO:crash_reporting.cc(215)] Crash reporting enabled for process: gpu-process
[0820/191019.165:INFO:crash_reporting.cc(215)] Crash reporting enabled for process: renderer
[0820/191020.399:ERROR:gpu_process_transport_factory.cc(1029)] Lost UI shared context.
[0820/191020.736:INFO:crash_reporting.cc(215)] Crash reporting enabled for process: renderer
apitrace: loaded into F:\ryan\.local\share\lutris\runners\winesteam\prefix\drive_c\Program Files\Steam\steamapps\common\Special Edition\DevilMayCry4SpecialEdition.exe
[0820/191022.444:WARNING:cert_verify_proc_win.cc(101)] Unknown error -2146762482 mapped to net::ERR_FAILED
apitrace: tracing to F:\ryan\.local\share\lutris\runners\winesteam\prefix\drive_c\Program Files\Steam\steamapps\common\Special Edition\DevilMayCry4SpecialEdition.2.trace
apitrace: warning: caught exception 0xc0000005
apitrace: flushing trace
wine: Unhandled page fault on read access to 0xd690e030 at address 0x17a45f0 (thread 00d4), starting debugger...

**Comment by AccountOneOff** [comment]
Try with Lutris master. I haven't tested as I don't have any DX10 games but Lutris doesn't currently work with DXVK's DX10.

I found this to work well with game launchers. I've done it lots of times with Lutris:

Download: https://people.freedesktop.org/~jrfonseca/apitrace/apitrace-msvc-latest.7z

1. Copy the apitrace lib folder to the game exe's parent folder
2. Copy all the files/folders from the apitrace bin folder and place them in the folder containing the game exe
3. Rename `apitrace.exe` to `game.exe` and `game.exe` to `game_o.exe`
4. Add `trace -a dxgi game_o.exe` to steam launch options

**Comment by romulasry** [comment]
I tried, same issue. Haven't tried lutris master, I will wait for an update.

**Comment by 3kinox** [comment]
Hello,

I have the same issue, but I managed to get an apitrace.
[DevilMayCry4_DX10.zip](https://github.com/doitsujin/dxvk/files/2315930/DevilMayCry4_DX10.zip)

**Comment by K0bin** [comment]
That trace is 3kb which means something went wrong while recording it.

**Comment by 3kinox** [comment]
Hmm I am no expert on apitrace, but the crash is almost immediate, so I don't know if it is much surprising that the trace be so small. After that, I did try all method I could find, and the one from  AccountOneOff is the one which gave me the trace.
But I do get a crash with trace as follows (now that I see it, I do have a doubt on whether dxvk is in use for proton dx10? Maybe I should post on proton issue...):
Unhandled exception: unimplemented function d3d11.dll.D3D11CoreCreateDevice called in 32-bit code (0x7bc541f1).
Register dump:
 CS:0023 SS:002b DS:002b ES:002b FS:0063 GS:006b
 EIP:7bc541f1 ESP:0033d268 EBP:0033d2cc EFLAGS:00000212(   - --  I   -A- - )
 EAX:7e1751fa EBX:7bce0ff4 ECX:0033d35c EDX:00000000
 ESI:0033d274 EDI:00211390
Stack dump:
0x0033d268:  0033d2d8 7bc5208a 7bc541ba 80000100
0x0033d278:  00000001 00000000 7bc541f1 00000002
0x0033d288:  7e17522c 7e1751fa f7f0eff4 00000003
0x0033d298:  0033d2c8 f7d71250 7e175604 0128c030
0x0033d2a8:  0112efe1 16c31e34 0033d2dc 00000001
0x0033d2b8:  f7d7123b 7e174ff4 001f9970 7e174ff4
Backtrace:
=>0 0x7bc541f1 stub_entry_point+0x51(dll="d3d11.dll", name="D3D11CoreCreateDevice", ret_addr=0x7e17173f) [/home/proton/proton/wine/dlls/ntdll/loader.c:222] in ntdll (0x0033d2cc)
  1 0x0034000f (0x0033d338)
  2 0x7e1892d5 D3D10CreateDevice+0x104(adapter=<couldn't compute location>, driver_type=<couldn't compute location>, swrast=<couldn't compute location>, flags=<couldn't compute location>, sdk_version=<couldn't compute location>, device=<couldn't compute location>) [/home/proton/proton/wine/dlls/d3d10/d3d10_main.c:126] in d3d10 (0x0033d3a8)
  3 0x008ee400 in devilmaycry4_dx10 (+0x4ee3ff) (0x7e180000)
  4 0x00000003 (0x00905a4d)
0x7bc541f1 stub_entry_point+0x51 [/home/proton/proton/wine/dlls/ntdll/loader.c:222] in ntdll: addl	$12,%esp
Unable to access file '/home/proton/proton/wine/dlls/ntdll/loader.c'
Modules:
Module	Address			Debug info	Name (136 modules)
PE	  400000-  f42000	Export          devilmaycry4_dx10
PE	  f50000- 12d1000	Deferred        dxgitrace
PE	10000000-10007000	Deferred        d3d11
PE	16f70000-16fe9000	Deferred        openvr_api_dxvk
PE	6f400000-6f596000	Deferred        dxgi
ELF	72a43000-736f0000	Deferred        libnvidia-glvkspirv.so.396.51
ELF	736f0000-75442000	Deferred        libnvidia-glcore.so.396.51
ELF	75442000-7907a000	Deferred        libllvm-6.0.so
ELF	79e8a000-7b400000	Deferred        steamclient.so
ELF	7b400000-7b7ea000	Deferred        kernel32<elf>
  \-PE	7b410000-7b7ea000	\               kernel32
ELF	7bc00000-7bd0f000	Dwarf           ntdll<elf>
  \-PE	7bc10000-7bd0f000	\               ntdll
ELF	7c000000-7c004000	Deferred        <wine-loader>
ELF	7c961000-7ce44000	Deferred        libvulkan_intel.so
ELF	7cfd5000-7d0ef000	Deferred        libglx_nvidia.so.0
ELF	7d0ef000-7d116000	Deferred        libtinfo.so.6
ELF	7d116000-7d14d000	Deferred        libedit.so.0
ELF	7d176000-7d375000	Deferred        libvulkan_radeon.so
ELF	7d375000-7d37e000	Deferred        libffi.so.6
ELF	7d3f3000-7d40a000	Deferred        steamoverlayvulkanlayer.so
ELF	7d40a000-7d40f000	Deferred        libnvidia-tls.so.396.51
ELF	7d40f000-7d41c000	Deferred        libdrm_amdgpu.so.1
ELF	7d41c000-7d438000	Deferred        libelf.so.1
ELF	7d438000-7d446000	Deferred        libwayland-client.so.0
ELF	7d446000-7d449000	Deferred        libxshmfence.so.1
ELF	7d449000-7d451000	Deferred        libxcb-sync.so.1
ELF	7d451000-7d45a000	Deferred        libxcb-xfixes.so.0
ELF	7d45a000-7d460000	Deferred        libxcb-dri3.so.0
ELF	7d460000-7d474000	Deferred        libdrm.so.2
ELF	7d474000-7d52c000	Deferred        msvcrt<elf>
  \-PE	7d490000-7d52c000	\               msvcrt
ELF	7d52c000-7d57a000	Deferred        libvulkan.so.1
ELF	7d57a000-7d5b9000	Deferred        winevulkan<elf>
  \-PE	7d580000-7d5b9000	\               winevulkan
ELF	7d5b9000-7d5d1000	Deferred        vulkan-1<elf>
  \-PE	7d5c0000-7d5d1000	\               vulkan-1
ELF	7d5d1000-7d5e5000	Deferred        api-ms-win-core-localization-obsolete-l1-2-0<elf>
  \-PE	7d5e0000-7d5e5000	\               api-ms-win-core-localization-obsolete-l1-2-0
ELF	7d5e5000-7d5f9000	Deferred        api-ms-win-core-datetime-l1-1-1<elf>
  \-PE	7d5f0000-7d5f9000	\               api-ms-win-core-datetime-l1-1-1
ELF	7d5f9000-7d60d000	Deferred        api-ms-win-core-string-l1-1-0<elf>
  \-PE	7d600000-7d60d000	\               api-ms-win-core-string-l1-1-0
ELF	7d60d000-7d81d000	Deferred        lsteamclient<elf>
  \-PE	7d6d0000-7d81d000	\               lsteamclient
ELF	7d81e000-7d822000	Deferred        libxcb-present.so.0
ELF	7d822000-7d825000	Deferred        libx11-xcb.so.1
ELF	7d825000-7d851000	Deferred        libpng12.so.0
ELF	7d897000-7d8ce000	Deferred        uxtheme<elf>
  \-PE	7d8a0000-7d8ce000	\               uxtheme
ELF	7d8d0000-7d8d7000	Deferred        libxfixes.so.3
ELF	7d8d7000-7d8e3000	Deferred        libxcursor.so.1
ELF	7d8e3000-7d8ef000	Deferred        libxrender.so.1
ELF	7d8ef000-7d903000	Deferred        libxi.so.6
ELF	7d903000-7d907000	Deferred        libxcomposite.so.1
ELF	7d907000-7d90e000	Deferred        libxxf86vm.so.1
ELF	7d90e000-7d912000	Deferred        libxinerama.so.1
ELF	7d912000-7d9a4000	Deferred        winex11<elf>
  \-PE	7d920000-7d9a4000	\               winex11
ELF	7d9a4000-7d9b8000	Deferred        api-ms-win-core-localization-l1-2-1<elf>
  \-PE	7d9b0000-7d9b8000	\               api-ms-win-core-localization-l1-2-1
ELF	7d9b8000-7d9cc000	Deferred        api-ms-win-core-fibers-l1-1-1<elf>
  \-PE	7d9c0000-7d9cc000	\               api-ms-win-core-fibers-l1-1-1
ELF	7d9cc000-7d9e0000	Deferred        api-ms-win-core-synch-l1-2-0<elf>
  \-PE	7d9d0000-7d9e0000	\               api-ms-win-core-synch-l1-2-0
ELF	7db7f000-7db88000	Deferred        libuuid.so.1
ELF	7db88000-7dbc3000	Deferred        libexpat.so.1
ELF	7dbc3000-7dc0d000	Deferred        libfontconfig.so.1
ELF	7dc0d000-7dc26000	Deferred        libz.so.1
ELF	7dc26000-7dc62000	Deferred        libpng16.so.16
ELF	7dc62000-7dc74000	Deferred        libbz2.so.1
ELF	7dc74000-7dd33000	Deferred        libfreetype.so.6
ELF	7dd33000-7de63000	Deferred        oleaut32<elf>
  \-PE	7dd50000-7de63000	\               oleaut32
ELF	7de63000-7de87000	Deferred        imm32<elf>
  \-PE	7de70000-7de87000	\               imm32
ELF	7de87000-7decc000	Deferred        usp10<elf>
  \-PE	7de90000-7decc000	\               usp10
ELF	7decc000-7e002000	Deferred        comctl32<elf>
  \-PE	7ded0000-7e002000	\               comctl32
ELF	7e002000-7e02d000	Deferred        msvfw32<elf>
  \-PE	7e010000-7e02d000	\               msvfw32
ELF	7e02d000-7e07b000	Deferred        avifil32<elf>
  \-PE	7e030000-7e07b000	\               avifil32
ELF	7e07b000-7e0c6000	Deferred        dsound<elf>
  \-PE	7e080000-7e0c6000	\               dsound
ELF	7e0c6000-7e0ec000	Deferred        wmvcore<elf>
  \-PE	7e0d0000-7e0ec000	\               wmvcore
ELF	7e0ec000-7e160000	Deferred        d3dcompiler_43<elf>
  \-PE	7e100000-7e160000	\               d3dcompiler_43
ELF	7e160000-7e176000	Deferred        d3d10core<elf>
  \-PE	7e170000-7e176000	\               d3d10core
ELF	7e176000-7e1b5000	Dwarf           d3d10<elf>
  \-PE	7e180000-7e1b5000	\               d3d10
ELF	7e1b5000-7e1cc000	Deferred        xinput1_3<elf>
  \-PE	7e1c0000-7e1cc000	\               xinput1_3
ELF	7e1cc000-7e1e8000	Deferred        dinput8<elf>
  \-PE	7e1d0000-7e1e8000	\               dinput8
ELF	7e1e8000-7e213000	Deferred        msacm32<elf>
  \-PE	7e1f0000-7e213000	\               msacm32
ELF	7e213000-7e293000	Deferred        rpcrt4<elf>
  \-PE	7e220000-7e293000	\               rpcrt4
ELF	7e293000-7e3ea000	Deferred        ole32<elf>
  \-PE	7e2b0000-7e3ea000	\               ole32
ELF	7e3ea000-7e4a2000	Deferred        winmm<elf>
  \-PE	7e3f0000-7e4a2000	\               winmm
ELF	7e4a2000-7e517000	Deferred        shlwapi<elf>
  \-PE	7e4b0000-7e517000	\               shlwapi
ELF	7e517000-7e7b7000	Deferred        shell32<elf>
  \-PE	7e530000-7e7b7000	\               shell32
ELF	7e7b7000-7e7d1000	Deferred        version<elf>
  \-PE	7e7c0000-7e7d1000	\               version
ELF	7e7d1000-7e849000	Deferred        advapi32<elf>
  \-PE	7e7e0000-7e849000	\               advapi32
ELF	7e849000-7e976000	Deferred        gdi32<elf>
  \-PE	7e860000-7e976000	\               gdi32
ELF	7e976000-7eb74000	Deferred        user32<elf>
  \-PE	7e990000-7eb74000	\               user32
ELF	7efec000-7f000000	Deferred        libnss_files.so.2
ELF	f7665000-f766a000	Deferred        libxau.so.6
ELF	f766c000-f7699000	Deferred        libxcb.so.1
ELF	f7699000-f76b7000	Deferred        libgcc_s.so.1
ELF	f76b7000-f7718000	Deferred        libgldispatch.so.0
ELF	f7718000-f772e000	Deferred        libxext.so.6
ELF	f772e000-f787f000	Deferred        libx11.so.6
ELF	f787f000-f78a2000	Deferred        libglx.so.0
ELF	f78a2000-f79a4000	Deferred        libm.so.6
ELF	f7b34000-f7b39000	Deferred        libdl.so.2
ELF	f7b39000-f7b43000	Deferred        librt.so.1
ELF	f7b45000-f7ba9000	Deferred        libgl.so.1
ELF	f7ba9000-f7d4d000	Deferred        libc.so.6
ELF	f7d4d000-f7d6c000	Deferred        libpthread.so.0
ELF	f7d6c000-f7f23000	Dwarf           libwine.so.1
ELF	f7f23000-f7f5f000	Deferred        gameoverlayrenderer.so
ELF	f7f61000-f7f8a000	Deferred        ld-linux.so.2
ELF	f7f8d000-f7f8f000	Deferred        [vdso].so
Threads:
process  tid      prio (all id:s are in hex)
0000000c services.exe
	00000027    0
	00000021    0
	0000001c    0
	00000016    0
	00000013    0
	00000012    0
	00000011    0
	0000000e    0
	0000000d    0
0000000f winedevice.exe
	00000019    0
	00000018    0
	00000017    0
	00000015    0
	00000014    0
	00000010    0
0000001a plugplay.exe
	0000001e    0
	0000001d    0
	0000001b    0
0000001f winedevice.exe
	00000029    0
	00000028    0
	00000026    0
	00000025    0
	00000024    0
	00000023    0
	00000022    0
	00000020    0
0000002a (D) Z:\home\antoine\.local\share\Steam\steamapps\common\Devil May Cry 4\DevilMayCry4_DX10.exe
	00000034    0
	00000033    0
	00000032    0
	00000031    0
	0000002b    0 <==
0000002c explorer.exe
	00000030    0
	0000002f    0
	0000002e    0
	0000002d    0
System information:
    Wine build: wine-3.7
    Platform: i386 (WOW64)
    Version: Windows 7
    Host system: Linux
    Host version: 4.17.14-202.fc28.x86_64

**Comment by 3kinox** [comment]
Okay, after investigation the bug happens without dxvk anyway, so I guess trace or not is not important here... Will report to proton/wine instead.

**Comment by romulasry** [comment]
Did you run winetricks d3dx10_43 -- first?

**Comment by doitsujin** [comment]
Please apply the following patch to DXVK, re-run the game and post the resulting log files. It won't solve any problems, but rather add a more useful error message. It appears that it uses an image format that DXVK currently doesn't support.

[dxvk-texture-log.patch.txt](https://github.com/doitsujin/dxvk/files/2320182/dxvk-texture-log.patch.txt)

**Comment by 3kinox** [comment]
Here goes the log, with your patch.
But with every new prefix I create, I can no longer obtain that same error... Only on a steam prefix I reused and with Steam Play/proton...
Instead there I get a good old page fault... (I joined the terminal log too). I also no longer can get an apitrace, I suppose the pf kills apitrace before the dump...
Guess I will try to make a dump with reinstalling patched dxvk on my other prefix tomorrow morning too.
[dxvk-log.zip](https://github.com/doitsujin/dxvk/files/2320981/dxvk-log.zip)

**Comment by 3kinox** [comment]
Here is the log I get, with patched dxvk and the right error. I don't see any sign of your new exception, but now the wine debugger fail and I can 't get a full backtrace.  

DXVK log:

1 info:  Game: DevilMayCry4_DX10.exe
  2 info:  DXVK: v0.70-8-g73cbf5b
  3 warn:  OpenVR: Failed to locate module
  4 info:  Enabled instance extensions:
  5 info:    VK_KHR_get_physical_device_properties2
  6 info:    VK_KHR_surface
  7 info:    VK_KHR_win32_surface
  8 info:  GeForce GTX 1070:
  9 info:    Driver: 396.51.0
 10 info:    Vulkan: 1.1.70
 11 info:    Memory Heap[0]:
 12 info:      Size: 8192 MiB
 13 info:      Flags: 0x1
 14 info:      Memory Type[7]: Property Flags = 0x1
 15 info:      Memory Type[8]: Property Flags = 0x1
 16 info:    Memory Heap[1]:
 17 info:      Size: 12016 MiB
 18 info:      Flags: 0x0
 19 info:      Memory Type[0]: Property Flags = 0x0
 20 info:      Memory Type[1]: Property Flags = 0x0
 21 info:      Memory Type[2]: Property Flags = 0x0
 22 info:      Memory Type[3]: Property Flags = 0x0
 23 info:      Memory Type[4]: Property Flags = 0x0
 24 info:      Memory Type[5]: Property Flags = 0x0
 25 info:      Memory Type[6]: Property Flags = 0x0
 26 info:      Memory Type[9]: Property Flags = 0x6
 27 info:      Memory Type[10]: Property Flags = 0xe
 28 info:  GeForce GTX 1070:
 29 info:    Driver: 396.51.0
 30 info:    Vulkan: 1.1.70
 31 info:    Memory Heap[0]:
 32 info:      Size: 8192 MiB
 33 info:      Flags: 0x1
 34 info:      Memory Type[7]: Property Flags = 0x1
 35 info:      Memory Type[8]: Property Flags = 0x1
 36 info:    Memory Heap[1]:
 37 info:      Size: 12016 MiB
 38 info:      Flags: 0x0
 39 info:      Memory Type[0]: Property Flags = 0x0
 40 info:      Memory Type[1]: Property Flags = 0x0
 41 info:      Memory Type[2]: Property Flags = 0x0
 42 info:      Memory Type[3]: Property Flags = 0x0
 43 info:      Memory Type[4]: Property Flags = 0x0
 44 info:      Memory Type[5]: Property Flags = 0x0
 45 info:      Memory Type[6]: Property Flags = 0x0
 46 info:      Memory Type[9]: Property Flags = 0x6
 47 info:      Memory Type[10]: Property Flags = 0xe
 48 warn:  DxgiFactory::QueryInterface: Unknown interface query
 49 warn:  7632e1f5-ee65-4dca-87fd-84cd75f8838d
 50 warn:  DxgiFactory::QueryInterface: Unknown interface query
 51 warn:  1bc6ea02-ef36-464f-bf0c-21ca39e5168a
 52 warn:  DxgiFactory::QueryInterface: Unknown interface query
 53 warn:  25483823-cd46-4c7d-86ca-47aa95b837bd
 54 warn:  DxgiAdapter::QueryInterface: Unknown interface query
 55 warn:  645967a4-1392-4310-a798-8053ce3e93fd

Relevant wine log:
wine: Call from 0x7bc79441 to unimplemented function D3D11.DLL.D3D11CoreCreateDevice@24, aborting
apitrace: warning: caught exception 0x80000100
apitrace: flushing trace
wine: Unimplemented function D3D11.DLL.D3D11CoreCreateDevice@24 called at address 0x7bc79441 (thread 0086), starting debugger...

If i remove the apitrace.exe trick from AccountOneOff, I get a page fault. With apitrace.exe I get this. On pure wine with clean prefix everything works in any case (apitrace, no apitrace). With dxvk prefix but with "WINEDLLOVERRIDES="d3d11,dxgi=b" wine game.exe" the game still fails in the same way.
And in proton it fails in the same way, but I get the full backtrace.

**Comment by romulasry** [comment]
Any update?

**Comment by romulasry** [comment]
Is dxvk-texture-log.patch.txt in the latest dxvk (0.80)? I am waiting on a fix for lutris if so I may send a log.

**Comment by romulasry** [comment]
It stars now (latest dxvk 0.80) under Proton but crashes when I try to run the frame rate test:
[steam-329050.log](https://github.com/doitsujin/dxvk/files/2430061/steam-329050.log)

**Comment by pchome** [comment]
@romulasry 
AFAIK Proton currently not using d3d10 libs from DXVK.

**Comment by romulasry** [comment]
> @romulasry
> AFAIK Proton currently not using d3d10 libs from DXVK.

Darn... :(

**Comment by romulasry** [comment]
Here is a log from the 0.81:

[dxvk.log](https://github.com/doitsujin/dxvk/files/2452828/dxvk.log)

**Comment by Vosester** [comment]
With the latest master, I have been playing with memory sizes, the best I got is setting this. It runs just past the Capcom logo then dies.

dxgi.maxDeviceMemory = 2048
dxgi.maxSharedMemory = 2048

Anyone tried it in a 32bit prefix?, Wine's WOW64 could be the problem or it's just a 32bit app been flaky.
I tried to get an api trace but it was pain to set up and it failed. 
Could we get an api dump within DXVK as a build option, to make make it easier to debug?

**Comment by jas99** [comment]
+1

**Comment by massatt212** [comment]
OS: Windows 7, Windows 8.1
    Processor: Intel(R) Core(TM) 2 Duo 2.4GHz, AMD Athlon(TM) X2 2.8 GHz
    Memory: 2 GB RAM
    Graphics: NVIDIA(R) GeForce(R) 8800 GTS, AMD Radeon(TM) HD 3850 with 512MB RAM or better
    DirectX: Version 10
    Storage: 27 GB available space
    Sound Card: DirectSound Support (DirectX® 9.0c)
    Additional Notes: Some high end integrated graphics and modern gaming laptops with a discrete GPU may work but have not been tested, nor are they officially supported by Capcom.

But its trying to run DX11
before the DXVK dx10 release it tries to boot with the 11

**Comment by massatt212** [comment]
the stupid game starts with DX11 and then goes to DX10
well done capcom smh cant even get it working on Windows DXVK just errors when trying to swap over to dx10

**Comment by massatt212** [comment]
Devil may cry 4 special edition works
0.95 thx for the update will report if it crashes (i mean just report lol)
